python/src/OTtestcode_doc.i.in

```diff
@@ -0,0 +1,188 @@
+%feature("docstring") OT::Test::assert_almost_equal
+"Raises an exception if two objects are not equal up to desired tolerance.
+
+Similar to numpy.testing.assert_allclose.
+
+Parameters
+----------
+a, b : float, Point, Matrix, Distribution, Mesh, Field or ProcessSample
+    Elements to compare
+rtol : float
+    Relative tolerance, default is 1e-5
+atol : float
+    Absolute tolerance, default is 1e-8
+errMsg : str
+    Error message emitted when comparison fails
+
+Examples
+--------
+>>> import openturns as ot
+>>> import openturns.testing as ott
+>>> dist = ot.Uniform(0.0, 1.0)
+>>> ott.assert_almost_equal(dist.getMean(), [0.5])
+"
+
+// ---------------------------------------------------------------------
+    
+%feature("docstring") OT::Test::DistributionChecker
+"Automates distribution tests.
+
+It helps performing tests to validate :class:`~openturns.Distribution` services:
+
+- Comparison operators
+- PDF: analytical PDF vs finite-difference estimate from CDF
+- Log PDF: verify consistency of PDF and LogPDF values
+- DDF: analytical DDF vs finite-difference estimate from PDF
+- CCDF: verify consistency between CDF and complementary CDF
+- Survival function: verify consistency between survival and CDF values
+- Inverse survival function: verify consistency between inverse survival and survival
+- Quantile function: verify consistency between quantile and CDF values
+- Probability of an interval: verify probabilities of the range, full numerical range
+- Sampling: Pass Kolmogorov or Chi2 test for different sampling sizes
+- Entropy test: verify consistency between analytical entropy and empirical entropy
+- Moments: verify consistency between analytical moments and empirical moments
+- Parameters: verify setParameter/getParameter accessors consistency
+- Gradients: assess PDF/CDF analytical gradients consistency against their finite difference estimates
+- Minimum volume interval: verify the probability of the interval
+- Minimum volume level-set: verify the probability of the level-set
+- Confidence interval: verify the probability of the level-set
+- Characteristic function
+
+Parameters
+----------
+distribution : :class:`~openturns.Distribution`
+    The distribution instance to test.
+
+Examples
+--------
+>>> import openturns as ot
+>>> import openturns.testing as ott
+>>> dist = ot.Uniform(0.0, 1.0)
+>>> checker = ott.DistributionChecker(dist)
+>>> ot.Log.Show(ot.Log.TRACE)  # doctest: +SKIP
+>>> checker.run()  # doctest: +SKIP
+"
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::Test::DistributionChecker::run
```

Comment:
There are two undocumented methods `setEntropySamplingSize` and `setEntropyTolerance`:

![image](https://github.com/user-attachments/assets/86e87d47-ab04-4d94-8cea-47ea3163ff74)